modules/cnf-image-based-upgrade-share-container-directory.adoc

```diff
@@ -2,15 +2,15 @@
 // * scalability_and_performance/ztp-image-based-upgrade.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="ztp-image-based-upgrade-shared-container-directory_{context}"]
+[id="cnf-image-based-upgrade-shared-container-directory_{context}"]
 = Sharing the container directory between `ostree` stateroots
 
 You must apply a `MachineConfig` to both the seed and the target clusters during installation time to create a separate partition and share the `/var/lib/containers` directory between the two `ostree` stateroots that will be used during the upgrade process.
 
-[id="ztp-image-based-upgrade-shared-container-directory-acm_{context}"]
-== Sharing the container directory between `ostree` stateroots when using {rh-rhacm}
+[id="cnf-image-based-upgrade-shared-container-directory-acm_{context}"]
+== Sharing the container directory between `ostree` stateroots
```

Comment:
Why is this secondary title repeated? 
![image](https://github.com/openshift/openshift-docs/assets/74046732/0019f321-5eba-4d59-87c4-a0c4a226dea3)

Reply:
Suggest dropping the top "Sharing the container directory between ostree stateroots" section in favour of two equal sections, one for ZTP, one for manual upgrade use case.